operator/pkg/ciliumendpointslice/endpointslice.go

```diff
@@ -375,3 +381,26 @@ func ciliumEndpointSliceInit(client csv2a1.CiliumV2alpha1Interface, ctx context.
 	cache.WaitForCacheSync(ctx.Done(), cesController.HasSynced)
 	return cesStore
 }
+
+// UsedIdentitiesInCESs returns all Identities that are used in CESs.
+func UsedIdentitiesInCESs(cesStore *cache.Store) map[string]bool {
```

Comment:
Why this methods get a store? Couldn't it just use the store from vars?

Reply:
You are right. I fixed it. The CES store doesn't need to be exported.